tests/test_laser_alignment.py

```diff
@@ -0,0 +1,98 @@
+# This file is part of ts_audio_trigger.
+#
+# Developed for the Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+import pathlib
+import unittest
+from typing import TypeAlias
+
+import numpy as np
+from lsst.ts.audiotrigger import LaserAlignmentListener, Relay
+
+logging.basicConfig(
+    format="%(asctime)s:%(levelname)s:%(name)s:%(message)s", level=logging.DEBUG
+)
+
+PathT: TypeAlias = str | pathlib.Path
+
+# Standard timeout in seconds
+TIMEOUT = 5
+
+
+class LaserAlignmentTestCase(unittest.IsolatedAsyncioTestCase):
+    def setUp(self) -> None:
+        self.log = logging.getLogger()
+        self.data_dir = pathlib.Path(__file__).parent / "data" / "config"
+
+    @unittest.skip("Not sure how to mock yet.")
```

Comment:
Do you intend to fix this in this ticket? I may be able to help you with that. Otherwise please open a Jira ticket and add a TODO here.

Reply:
You marked this comment as resolved but no reference to a DM ticket was added. Again, open a DM ticket an add the ticket ID here.